app/common/src/services/Backend.ts

```diff
@@ -1433,13 +1433,23 @@ export function compareAssets(a: AnyAsset, b: AnyAsset) {
 
   if (relativeTypeOrder !== 0) {
     return relativeTypeOrder
+  } else {
+    // We sort by modified date, because the running/recent projects should be at the top.
+    // but above the all folders
+    const aModified = Number(new Date(a.modifiedAt))
+    const bModified = Number(new Date(b.modifiedAt))
+    const modifiedDelta = aModified - bModified
+    if (modifiedDelta !== 0) {
+      // Sort by date descending, rather than ascending.
+      return -modifiedDelta
+    } else {
+      return (
+        a.title > b.title ? 1
```

Comment:
probably we need to lowercase the title before comparing them

Reply:
we can use `localeCompare` for case insensitive compare